source/common/http/http1/codec_impl.cc

```diff
@@ -883,6 +887,7 @@ StatusOr<CallbackResult> ConnectionImpl::onHeadersCompleteImpl() {
           "http/1.1 protocol error: both 'Content-Length' and 'Transfer-Encoding' are set.");
     }
   }
+#endif
```

Comment:
I forgot it needs to go to line 905 since transfer-encoding validation also moved to UHV